apps/ledger-live-mobile/src/newArch/features/LandingPages/screens/GenericLandingPage/index.tsx

```diff
@@ -18,7 +21,23 @@ type NavigationProps = BaseComposite<
 const GenericLandingPage = (props: NavigationProps) => {
   const useCase = props.route.params?.useCase;
   const isLoading = useSelector(isDynamicContentLoadingSelector);
-  const { categoriesCards } = useDynamicContent();
+  const { categoriesCards, getStickyCtaCardByLandingPage, trackContentCardEvent } =
```

Comment:
Following the new arch conventions, we might have a useViewModel hook sibling to the screen where we can put everything related to the data. So we can move the useSelector, and useDynamicContent directly inside this hook.
Then can you create a unit test to test this screen, please?